harness/determined/common/experimental/user.py

```diff
@@ -82,6 +82,7 @@ def change_display_name(self, display_name: str) -> None:
         self.reload()
 
     def change_password(self, new_password: str) -> None:
+        api.check_password_complexity(new_password)
```

Comment:
Since this is a user-facing function, I think it would be friendly to tell the user in a docstring that bad passwords will result in a `ValueError`. Makes it a little easier for a user to handle this function programmatically without having to read into its implementation.

If you do add a docstring, could you please do so following [our style guide](https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1505230999/Python+Code+Style#Rule-exceptions) / Google docstring style?